nixos/modules/services/networking/prosody.nix

```diff
@@ -1,885 +1,659 @@
-{ config, lib, pkgs, ... }:
-
-with lib;
+{ options, config, pkgs, lib, ... }:
 let
-  cfg = config.services.prosody;
+  inherit (lib) mkChangedOptionModule mkDefault mkEnableOption mkForce mkIf mkMerge mkOption mkRenamedOptionModule;
+  inherit (lib) all any attrValues boolToString concatStringsSep filterAttrs genAttrs hasPrefix listToAttrs literalExpression mapAttrsToList mdDoc optionals optionalAttrs optionalString types unique;
```

Comment:
IMO I find those lists worse than ``with lib;``.

Reply:
Yeah I'm not sure what to say here. Apparently there is a push to stop using `with lib;` I'm told, and in principle agree with... but this solution is also understandable because without a great LSP server to remind you this ends up with cruft after time. Using `lib.` everywhere is super annoying.

What is your preference?

Reply:
I usually just opt to write `lib.all`, `lib.any`, inline anymore.

Reply:
oof